app/src/test/java/com/greenfox/opal/gitinder/RobolectricTest.java

```diff
@@ -1,30 +1,27 @@
 package com.greenfox.opal.gitinder;
 
-import android.widget.Button;
-import android.widget.EditText;
-import android.widget.TextView;
-
+import android.content.Intent;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.robolectric.Robolectric;
 import org.robolectric.RobolectricTestRunner;
 import org.robolectric.annotation.Config;
 
 import static org.junit.Assert.assertEquals;
+import static org.robolectric.Shadows.shadowOf;
 
 @RunWith(RobolectricTestRunner.class)
 @Config(constants = BuildConfig.class)
 public class RobolectricTest {
 
-    @Test
-    public void wildButtonTest() throws Exception {
-        MainActivity activity = Robolectric.setupActivity(MainActivity.class);
+  MainActivity mainActivity = Robolectric.setupActivity(MainActivity.class);
+  LoginActivity loginActivity = Robolectric.setupActivity(LoginActivity.class);
 
-        Button button = (Button)activity.findViewById(R.id.button);
-        TextView result = (TextView)activity.findViewById(R.id.textView);
-        EditText edit = (EditText)activity.findViewById(R.id.editText);
+  @Test
+  public void checkLoginNoUser() {
+    mainActivity.checkLogin();
```

Comment:
@balintvecsey you dont actually need to call the `checkLogin()` method, when the activity is created (by Roboelectric) it should call this